pkgs/tools/security/regpg/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, fetchgit
+, git
+, perl
+, perlTextMarkdown
+}:
+
+stdenv.mkDerivation rec {
+  pname = "regpg";
+  version = "1.11";
+
+  src = fetchgit {
+    url = "https://github.com/fanf2/regpg.git";
+    rev = "refs/tags/regpg-${version}";
+    sha256 = "9492edfa89d044c37a72f48e5753c0054fdf7f1100c834c8eadf518ce93d0d1c";
+    leaveDotGit = true;
+  };
+
+  builder = builtins.toFile "builder.sh" ''
```

Comment:
please use the default builder. It has all sorts of niceties that are easy to forget when making your own.
Add git and perl to the path with `nativeBuildInputs = [ git perl ]`
run patchShebang in postPatch and postBuild
etc